addons/event_sale/static/src/js/event_configurator_widget.js

```diff
@@ -1,59 +1,144 @@
+                        },
+                        dataPointID
+                    );
+                    return Promise.resolve(true);
+                }
+                return Promise.resolve(false);
+            });
+        }
+        return Promise.resolve(false);
+    },
+
+    /**
+     * Opens the event configurator in 'edit' mode.
+     *
+     * @override
+     * @private
+     */
+    _onEditProductConfiguration: function () {
+        if (this.recordData.event_ok) {
+            var defaultValues = {
+                default_product_id: this.recordData.product_id.data.id
+            };
 
-        this._super.apply(this, arguments);
-        if (!event.data.changes.product_id){
-            return;
+            if (this.recordData.event_id) {
+                defaultValues['default_event_id'] = this.recordData.event_id.data.id;
+            }
+
+            if (this.recordData.event_ticket_id) {
+                defaultValues['default_event_ticket_id'] = this.recordData.event_ticket_id.data.id;
+            }
+
+            this._openEventConfigurator(defaultValues, this.dataPointID);
+        } else {
+            this._super.apply(this, arguments);
         }
+    },
 
-        var productId = event.data.changes.product_id.id;
-        self._checkForEvent(productId, event.data.dataPointID);
+    /**
+     * Opens the event configurator to allow configuring the SO line with events information.
+     *
+     * When the window is closed, configured values are used to trigger a 'field_changed'
+     * event to modify the current SO line.
+     *
+     * If the window is closed without providing the required values 'event_id' and
+     * 'event_ticket_id', the product_id field is cleaned.
+     *
+     * @param {Object} data various "default_" values
+     * @param {string} dataPointId
+     *
+     * @private
+     */
+    _openEventConfigurator: function (data, dataPointId) {
+        var self = this;
+        this.do_action('event_sale.event_configurator_action', {
+            additional_context: data,
+            on_close: function (result) {
+                if (result && result !== 'special'){
+                    self.trigger_up('field_changed', {
+                        dataPointID: dataPointId,
+                        changes: result.eventConfiguration,
+                        // Do we unselect the row after event config?
+                        // event_sale tour is made to work without unselectRow
+                        /*onSuccess: function (){
+                            self.getParent().unselectRow();
+                        }*/
+                    });
+                } else {
+                    if (!self.recordData.event_id || !self.recordData.event_ticket_id) {
+                        self.trigger_up('field_changed', {
+                            dataPointID: dataPointId,
+                            changes: {product_id: {operation: 'DELETE_ALL'}},
```

Comment:
hu? Operations are for One2Many and Many2Many fields, not Many2one. I guess you want to unset the field? Then `product_id: false`.